src/components/Header/Header.js

```diff
@@ -59,12 +61,26 @@ class Header extends Component {
         }
         this.handleRightMenuClick = this.handleRightMenuClick.bind(this); 
         this.handleBoardMenuClick = this.handleBoardMenuClick.bind(this);
-        this.handleHeader = this.handleHeader.bind(this);
         this.closeMenus = this.closeMenus.bind(this);
         this.openFeedbackModal = this.openFeedbackModal.bind(this);
         this.closeFeedbackModal = this.closeFeedbackModal.bind(this);
     }
 
+    componentWillMount(){
+        const useridForSideBar = 1;
+        // const useridForSideBar = this.props.match.params.userid;
+        findDashboardInfo(useridForSideBar)
+            .then(res => {
+                if (res.status !== 200){
+                    console.log(res);
+                }
+                else {
+                    this.setState(res.data)
```

Comment:
Landon, I think I have the answer to your question.

When the component mounts, it sets the ENTIRE state of the component to just what it receives from the promise. Which looks like this:

```
{
  groups: [
        { id: 1, created_by: 2, name: "The Rebel Alliance", status_id: 1},
        { id: 2, created_by: 6, name: "The First Galactic Empire", status_id: 1 }
  ],
  projects: [
        { id: 1, name: "Defeat the Empire", author_id: 2, status_id: 1},
        { id: 2, name: "Destroy the Rebels", author_id: 6, status_id: 1}
  ]
}
```

So I think the solution to your problem would be to destructure the object:

`const { groups, projects } = res.data;`

And then set the individual state properties to those values:
`this.setState({ groups, projects });`

I hope this answers your question.